lesson4Task1.py

```diff
@@ -0,0 +1,43 @@
+# TASK-1
+# Write a function:
+# def solution(A)
+# that, given an array A of N integers,
+# returns the smallest positive integer (greater than 0) that does not occur in A.
+# For example, given A = [1, 3, 6, 4, 1, 2], the function should return 5.
+# Given A = [1, 2, 3], the function should return 4.
+# Given A = [−1, −3], the function should return 1.
+# Assume that:
+# N is an integer within the range [1..100,000];
+# each element of array A is an integer within the range [−1,000,000..1,000,000].
+
+
+def search_in_array(array):
```

Comment:
easy way

    for i in range(1, len(l) + 2):
        if i not in l:
            return i